src/structs/lepton_header.rs

```diff
@@ -205,6 +205,8 @@ impl LeptonHeader {
 
             let last = &mut self.thread_handoff[num_threads - 1];
 
+            let max_last_segment_size = max_last_segment_size;
```

Comment:
? :)

Reply:
oh yeah.... I'll fix that. I have a subsequent refactoring to split out the jpeg code from the lepton code I'll fix that there. Thanks!

Reply:
You can put it in this PR, it'll be OK.

Reply:
Fine :)